addons/popochiu/editor/popups/setup/setup.gd

```diff
@@ -54,21 +50,61 @@ func _ready() -> void:
 	# Set default state
 	template_description_container.hide()
 	template_description.text = ""
-	
-	hide()
 
 
 #endregion
 
 #region Public #####################################################################################
-func appear(show_welcome := false) -> void:
+func on_about_to_popup() -> void:
+	welcome.add_theme_font_override("bold_font", get_theme_font("bold", "EditorFonts"))
+	scale_message.add_theme_font_override("normal_font", get_theme_font("main", "EditorFonts"))
+	scale_message.add_theme_font_override("bold_font", get_theme_font("bold", "EditorFonts"))
+	scale_message.add_theme_font_override("mono_font", get_theme_font("doc_source", "EditorFonts"))
+	gui_templates_title.add_theme_font_override("font", get_theme_font("bold", "EditorFonts"))
+	gui_templates_description.add_theme_font_override(
+		"font", get_theme_font("doc_source", "EditorFonts")
+	)
+	template_description.add_theme_font_override("bold_font", get_theme_font("bold", "EditorFonts"))
+
+
+func on_close() -> void:
+	if _is_closing:
+		return
+	
+	_is_closing = true
+	
+	ProjectSettings.set_setting(PopochiuResources.DISPLAY_WIDTH, int(game_width.value))
+	ProjectSettings.set_setting(PopochiuResources.DISPLAY_HEIGHT, int(game_height.value))
+	ProjectSettings.set_setting(PopochiuResources.TEST_WIDTH, int(test_width.value))
+	ProjectSettings.set_setting(PopochiuResources.TEST_HEIGHT, int(test_height.value))
+	
+	match game_type.selected:
+		1:
```

Comment:
It would be better for those to be constants maybe?

Reply:
Yes sir.

Reply:
Done.